easybuild/easyconfigs/z/zlib/zlib-1.2.8-foss-2016.04.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'ConfigureMake'
+
+name = 'zlib'
+version = '1.2.8'
+
+homepage = 'http://www.zlib.net/'
+description = """zlib is designed to be a free, general-purpose, legally unencumbered
+ -- that is, not covered by any patents -- lossless data-compression library for use
+ on virtually any computer hardware and operating system."""
+
+toolchain = {'name': 'foss', 'version': '2016.04'}
+toolchainopts = {'pic': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = [('http://sourceforge.net/projects/libpng/files/zlib/%(version)s', 'download')]
+
+sanity_check_paths = {
+    'files': ['include/zconf.h', 'include/zlib.h', 'lib/libz.a', 'lib/libz.so'],
```

Comment:
@AFCJamie please use `'lib/libz.%s' % SHLIB_EXT` rather than `'lib/libz.so'`, like in the existing easyconfigs for   `zlib` 1.2.8